ep_gui/resources/lang/ep_gui.en-US.rc

```diff
@@ -362,16 +362,16 @@ BEGIN
     IDS_OTHER_PWRBTNACTION_2    "Shut down (default)"
     IDS_OTHER_PWRBTNACTION_4    "Restart"
     IDS_OTHER_NOREDIRECT        "Prevent the following Control Panel links from being redirected to the Settings app:"
-    IDS_OTHER_NOREDIRECT_SYSTEM "System/About page"
+    IDS_OTHER_NOREDIRECT_SYSTEM "System / About page"
     IDS_OTHER_NOREDIRECT_PROGRAMS "Programs && Features"
-    IDS_OTHER_NOREDIRECT_DATETIME "Adjust Date && Time"
-    IDS_OTHER_NOREDIRECT_TRAYICONS "Customize Notification-icons"
+    IDS_OTHER_NOREDIRECT_DATETIME "Adjust Date and Time"
+    IDS_OTHER_NOREDIRECT_TRAYICONS "Customize Notification icons"
 
     IDS_UPDATES                 "Updates"
 
     IDS_UPDATES_POLICY          "When File Explorer starts"
     IDS_UPDATES_POLICY_1        "Only notify when updates are available (default)"
-    IDS_UPDATES_POLICY_0        "Notify in all cases, updates available or no updates available"
+    IDS_UPDATES_POLICY_0        "Notify in all cases, presence or absence of update"

```

Comment:
"updates" instead of "update" in case there are multiple.

Reply:
oops! "updates" is more sensible and consistent with the top line. I correct it and put it back to reading. Greetings and best wishes 2024